_episodes/05-scripts.md

```diff
@@ -15,27 +15,45 @@ keypoints:
 
 ## How OpenRefine records what you have done
 
-As you conduct your data cleaning and preliminary analysis, OpenRefine saves every change you make to the dataset. These
-changes are saved in a format known as JSON (JavaScript Object Notation). You can export this JSON script and apply it to other data files. If you had 20 files to clean, and they all had the same type of errors (e.g. misspellings, leading white spaces), and all
-files had the same column names, you could save the JSON script, open a new file to clean in OpenRefine, paste in the script and run it.
-This gives you a quick way to clean all of your related data.
+As you conduct your data cleaning and preliminary analysis, OpenRefine saves
+every change you make to the dataset. These changes are saved in a format known
+as JSON (JavaScript Object Notation). You can export this JSON script and apply
+it to other data files. If you had 20 files to clean, and they all had
+the same type of errors (e.g. misspellings, leading white spaces), and all
+files had the same column names, you could save the JSON script, open a new
+file to clean in OpenRefine, paste in the script and run it. This gives you a
+quick way to clean all of your related data.
 
 ## Saving your work as a script
 
-1. In the `Undo / Redo` section, click `Extract...`, and select the steps that you want to apply to other datasets by clicking the check boxes.
+1. In the `Undo / Redo` section, click `Extract...`, and select the steps that
+   you want to apply to other datasets by clicking the check boxes.
 
-![History](../fig/history.png)
+   ![History](../fig/history.png)
```

Comment:
Great, this seems to fix an issue mentioned in #95 (which was closed already, without providing a fix...)